src/battle_ai_util.c

```diff
@@ -431,6 +431,7 @@ bool32 IsDamageMoveUnusable(u32 move, u32 battlerAtk, u32 battlerDef)
             return TRUE;
         break;
     case EFFECT_LOW_KICK:
+    case EFFECT_HEAT_CRASH:
```

Comment:
weird that this change is here since it is already in master

Reply:
I thought the same thing but didn't question it since it seemed harmless

Reply:
It's because this branch hasn't been synced to master after the other PR :)